docker/autoware-universe/Dockerfile

```
@@ -55,6 +55,15 @@ RUN mkdir -p /etc/OpenCL/vendors \
   && echo "libnvidia-opencl.so.1" > /etc/OpenCL/vendors/nvidia.icd \
   && chmod 644 /etc/OpenCL/vendors/nvidia.icd
 
+## Fix OpenGL issues (e.g. black screen in rviz2) due to old mesa lib in Ubuntu 22.04
+## See https://github.com/autowarefoundation/autoware.universe/issues/2789
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common \
```

If it's a tentative solution, how about adding a TODO comment like this?

```suggestion
TODO: Remove in Ubuntu 24.04
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common \
```